session/session.go

```diff
@@ -2396,6 +2397,7 @@ func (s *session) ExecutePreparedStmt(ctx context.Context, stmtID uint32, args [
 	} else {
 		is = s.GetInfoSchema().(infoschema.InfoSchema)
 	}
+	txnManager.SetStmtNode(preparedStmt.PreparedAst.Stmt)
```

Comment:
I think we can construct a `ExecuteStmt` and passed it to `TxnManager` instead of passing the inner statement